dockerfiles/zesty/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:precise
+FROM ubuntu:zesty
```

Comment:
o jesus... I haven't realized that it is EOL of the precise which served us for so long on travis...  looking at the recent travis logs -- it is still used!  I wondered if we could keep it at least until travis uses it?